### DISCUSSION TITLE
subgrain boundary projection on a pole figure

### DISCUSSION THREAD

**Discussion by amirtopaz** [mtex-toolbox/mtex#811] [Ask Anything]
Hello,
Is there a way to plot boundary trace projection on a pole figure like the following image? (from Hildyard et al., 2008)
![image](https://user-images.githubusercontent.com/62837798/108983915-0a02e300-7698-11eb-9fe2-a156c552891c.png)

**Comment by kilir** [comment]
Hi,
if you plot a vector3d, you can use the option `'circle'`, the same works for if you use use plotPDF.
```
v = vector3d([3 2 1 ])
plot(v)
hold on
plot(v,'circle','add2all','linewidth',2,'linecolor',[0.9 0.1 0.4],'linestyle',':')
hold off
```
Use `grains.boundary.direction` of the boundary you like to plot. Note that this will not give you an inclination information as depicted in your picture.


Cheers,
Rüdiger

**Reply by amirtopaz** [reply]
thanks Rüdiger,
Is there a way to add the inclination?

**Reply by kilir** [reply]
Do you know the inclination?

**Reply by amirtopaz** [reply]
yes.. I used the 'flat' plane and created a new curved one from it on illustrator as you can see here, but it took a while... it would be much easier to do it on matlab
![image](https://user-images.githubusercontent.com/62837798/108999939-b39f9f80-76ab-11eb-8211-b6ba78bc8ef9.png)

**Reply by kilir** [reply]
Then it's just as easy as defining the vector3d from your data of the boundary plane orientation and plot it using the option `'circle'`.

Cheers,
Rüdiger

**Reply by amirtopaz** [reply]
the interaction of the boundary with the primitive is set by the boundary orientation (the two lines out of the primitive).
I want to set the inclination by its interaction with the mean orientation of one of the axes (as can be seen in the image above)
Is that possible?

**Reply by kilir** [reply]
If I understand you correctly, the plane normal you are looking for is the cross product (use `cross()`) of the grain boundary trace (`grains.boundary.direction`) and mean of the directions parallel to some crystal direction ( e.g. `mean(orientation*Miller(0,1,0,cs))`)
Cheers,
Rüdiger
